tests/unit/multivariate/test_tree.py

```diff
@@ -655,18 +655,18 @@ def test_get_likelihood_no_parents(self, bivariate_mock):
 
         expected_partial_derivative_call_args = [
             (
-                (np.array([[
+                (np.array([[  # noqa: JS101
```

Comment:
@csala I'm not sure what to do about these. If I do follow the lint rule, I have to do the following:

```
(
  np.array(
    [
      [
        [0.25, 0.75],
        [0.50, 0.50],
      ]
    ]
  ),
), {}
```

which looks silly.

Reply:
Yeah it is a bit excessive, but my preference would be to do the formatting over ignoring the rule here.

Reply:
I agree, it's a bit excessive, and I also agree that in general it's better to re-format than to ignore the rule. It's also happening in only two places, so it's not that bad to have it fully expanded.

Reply:
Done. I also created intermediate variables for them because the `flake8-expression-complexity` addon complains about these lines in a different PR.